src/style.css

```
@@ -0,0 +1,31 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 22px;
+  flex-direction: column;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars_item {
+  height: 16px;
+  width: 16px;
+  margin: 0 4px 0 0;
+
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--1 .stars_item:nth-child(1),
```

```suggestion
.stars--1 .stars_item:nth-child(-n + 1),
```

Replace to make all code the same style

---

I have corrected the error